parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -492,11 +499,10 @@ runcmd:
 - sudo /bin/chown -R etcd:etcd /var/lib/etcddisk
 - systemctl stop etcd
 - sudo -u etcd rm -rf /var/lib/etcd/default
-- sudo /bin/sed -i s/Restart=on-abnormal/Restart=always/g /lib/systemd/system/etcd.service
```

Comment:
This is unrelated to TLS but removes an unnecessary line since /lib/systemd/system/etcd.service no longer exists